career.html

```diff
@@ -13,7 +13,7 @@ <h2>Join our international team and shape the future of 100Starlings and your ca
   <div class="container">
     {% for advert in site.pages %}
     {% if advert.path contains "career/" %}
-      <div class="career-adv"><a href="{{ advert.url }}">{{ advert.title }}</a></div>
+      <div onclick="location.href='{{ advert.url }}';" style="cursor: pointer;" class="career-adv"><a style="text-decoration: none;">{{ advert.title }}</a></div>
```

Comment:
Is the goal here to make the entire box clickable? I think it would be cleaner to achieve that from CSS or by swapping the `<a>` and the `<div>` tags making the anchor contain the div and not the other way around. Also, it would be preferable to put styling into CSS files instead of HTML attributes.